theme/scripts/productImagesSlider.ts

```diff
@@ -16,15 +16,12 @@ export default(function() {
 
 		setup(slider: HTMLElement) {
 			new Splide(slider, {
-				classes: {
```

Comment:
Removed because it seems like we no longer need these classes since we've set `arrows: false`